types/database/database.ts

```diff
@@ -59,7 +58,7 @@ export type OperationArgs = {
   transformer: (args: TransformerArgs) => Promise<Model>;
 };
 
-export type Models = Array<Class<Model>>;
+export type Models = Model[];
```

Comment:
Why this change?

Reply:
https://github.com/Nozbe/WatermelonDB/pull/1318 removed Class